tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/action/sdk/SDKCommandAction.java

```diff
@@ -74,7 +74,7 @@ protected IStatus run(IProgressMonitor monitor) {
 							SDK sdk = SDKUtil.getSDK(p);
 							ILiferayProject liferayProject = LiferayCore.create(ILiferayProject.class, p);
 
-							if (liferayProject != null) {
+							if ((liferayProject != null) && (sdk != null)) {
```

Comment:
hey @jtydhr88   can you help Seiphon review why in this case the sdk is null?  If we have a SDKCommandAction that has been enabled on a project where SDK is null.  We should never have an enabled SDKCommandAction for a project selection there SDKUtil.getSDK(p) would return null.